<launch>

    <rosparam command ="load" file="$(find Polhemus_Viper_ROS_Driver)/config/viper_sensor_startup_limit.yaml"/>
    <rosparam command ="load" file="$(find Polhemus_Viper_ROS_Driver)/config/viper_settings.yaml"/>
    <rosparam command ="load" file="$(find Polhemus_Viper_ROS_Driver)/config/ur_robot_limit.yaml"/>


    <node name = "viper_broadcaster" pkg = "Polhemus_Viper_ROS_Driver" type = "Viper_broadcaster" output = "screen"/>
    <node name = "kalman_filter" pkg = "Polhemus_Viper_ROS_Driver" type = "kalman_filter" output = "screen"/>
    <node name = "path_generation" pkg = "Polhemus_Viper_ROS_Driver" type = "path_generation" output = "screen"/>                  
    <node name = "outlier_detection" pkg = "Polhemus_Viper_ROS_Driver" type = "outlier_detection" output = "screen"/>    
</launch>